rust/perspective-vieux/package.json

```diff
@@ -17,7 +17,7 @@
         "src/**/*"
     ],
     "scripts": {
-        "build:wasm": "wasm-pack build --debug",
+        "build:wasm": "wasm-pack build --target web",
```

Comment:
We should explain this in the docs - end-users who import Perspective into their own Webpack builds will need to add the `arraybuffer-loader` in order to correctly load perspective.